stdlib/collections/__init__.pyi

```diff
@@ -247,23 +236,23 @@ class Counter(Dict[_T, int], Generic[_T]):
     def __iand__(self, other: Counter[_T]) -> Counter[_T]: ...
     def __ior__(self, other: Counter[_T]) -> Counter[_T]: ...  # type: ignore
 
-class _OrderedDictKeysView(KeysView[_KT], Reversible[_KT]):
+class _OrderedDictKeysView(_dict_keys[_KT, _VT], Reversible[_KT]):
     def __reversed__(self) -> Iterator[_KT]: ...
 
-class _OrderedDictItemsView(ItemsView[_KT, _VT], Reversible[Tuple[_KT, _VT]]):
+class _OrderedDictItemsView(_dict_items[_KT, _VT], Reversible[Tuple[_KT, _VT]]):
     def __reversed__(self) -> Iterator[Tuple[_KT, _VT]]: ...
 
-class _OrderedDictValuesView(ValuesView[_VT], Reversible[_VT]):
+class _OrderedDictValuesView(_dict_values[_VT, _KT], Reversible[_VT], Generic[_KT, _VT]):
```

Comment:
Should the generics here be covariant? the base types are.

Reply:
That makes sense: they are immutable, so a `_dict_items[int, str]` should also be a valid `_dict_items[object, object]`.

Nit: Please do not force-push or `git commit --amend`, so that we can see what changed after your previous review. We will squash the commits when merging, so the whole PR will appear as a single commit in the Git history anyway.